cli.js

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+'use strict';
+const meow = require('meow');
+const logSymbols = require('log-symbols');
+const np = require('./');
+
+const cli = meow(`
+	Usage
+	  $ np [patch | minor | major | <version>] (Default: patch)
+
+	Example
+	  $ np patch
+`);
+
+try {
+	np(cli.input[0]);
+} catch (err) {
+	console.log(` ${logSymbols.error} ${err.message}`);
```

Comment:
Should we call `process.exit(1)` here? If `np` is used in another CLI tool it should stop executing if something went wrong, no?